[EDU-4823] feat: add no-color flag (#1039)
aziontech/docs: src/content/docs/en/pages/devtools/cli/azion-cli/commands/globals/globals.mdx

The `h` or `--help` option displays more information about the `azion` command.

## no-color 

> The `--no-color` option indicates the output to not use colors.

Reviewer: ```suggestion
The --no-color option indicates that the output should not use colors.
```
